Week7/solution 7

```diff
@@ -0,0 +1,65 @@
+import pandas as pd
+
+#1: Add 12 Records. Take input from the user.
+data = []
+for _ in range(12):
+    prod_no = input("Enter Product Number: ")
+    prod_name = input("Enter Product Name: ")
+    sales = [int(input(f"Enter sales for {month}: ")) for month in ["Jan", "Feb", "Mar", "Apr", "May", "Jun"]]
+    data.append([prod_no, prod_name] + sales)
+
+#2: Create a DataFrame.
+df = pd.DataFrame(data, columns=["Prod_No", "Prod_Name", "Jan", "Feb", "Mar", "Apr", "May", "Jun"])
+
+#3: Change Column Names.
+df.columns = ["Product No", "Product Name", "January", "February", "March", "April", "May", "June"]
+
+#4: Add columns for "Total Sell" and "Average Sell".
+df["Total Sell"] = df.iloc[:, 2:].sum(axis=1)
+df["Average Sell"] = df.iloc[:, 2:8].mean(axis=1)
+
+#5: Add 2 rows at the end.
+new_rows = [["P13", "Product13", 1000, 2000, 3000, 4000, 5000, 6000, 21000, 3500],
+            ["P14", "Product14", 1500, 2500, 3500, 4500, 5500, 6500, 28000, 4666]]
+df = df.append(pd.DataFrame(new_rows, columns=df.columns), ignore_index=True)
+
+#6: Add 2 rows after the 3rd row.
+new_rows = [["P15", "Product15", 700, 800, 900, 1000, 1100, 1200, 5500, 916.67],
+            ["P16", "Product16", 800, 900, 1000, 1100, 1200, 1300, 6500, 1083.33]]
+df = pd.concat([df.iloc[:3], pd.DataFrame(new_rows, columns=df.columns), df.iloc[3:]]).reset_index(drop=True)
+
+#7: Print the first 5 rows.
+print("First 5 Rows:")
+print(df.head())
```

Comment:
what if i only need first two rows , how can you print it?

Reply:
print(df.head(2))